### PR TITLE
Add RMarkdown tutorial

### PR DESCRIPTION
**Pull request by mfakaehler**
I have added a tutorial that uses pica-rs in conjuction with R in a rmarkdown file. **Note, that the .md-file is automatically generated by the r-package knitr.** Any editing should be in the source-rmd rather then the .md-file. Before merging this into main, we need to figure out, which of the files need to be included:

- Do we need the original source-.rmd that generates the intermediate .md-file?
- Do we need to include an output html-file that a user would normally generate with pandoc from the attached rmd-file/ md-file?
- Do we include other files that are generated by knitr, for example the figures?

### CODE REVIEW

**Comment by niko2342** [comment]
@mfakaehler Thx for your contribution!

> * Do we need the original source-.rmd that generates the intermediate .md-file?

No.

> * Do we need to include an output html-file that a user would normally generate with pandoc from the attached rmd-file/ md-file?

No.

> * Do we include other files that are generated by knitr, for example the figures?

The figures are very useful to illustrate what is possible with R (Markdown).

I'm going to do a detailed review tomorrow.

File: docs/book/src/de/using-pica-rs-with-rmarkdown/using-pica-rs-with-rmarkdown.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+```
+
+Die Datei `title_idn_gnd_idn.csv` kann ggf. auch nur temporär erzeugt werden,
+wenn ein Zwischenspeichern des Outputs von `pica-rs`nicht gewünscht ist.
+Das Einlesen der Date in R erfolgt über die üblichen Funktionen, z.B. aus dem
+`tidyverse`-Unterpaket `readr`.
+
+
+```r
```
Comment (niko2342) [inline docs/book/src/de/using-pica-rs-with-rmarkdown/using-pica-rs-with-rmarkdown.md:38]:
```r
title_idn_gnd_idn <- read_csv("title_idn_gnd_idn.csv", 
    col_types = list(idn_titel = col_factor(), idn_sw = col_factor()))
```

File: docs/book/src/de/using-pica-rs-with-rmarkdown/using-pica-rs-with-rmarkdown.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+Wir wollen die abgefragten Daten im Folgenden beispielhaft analysieren:
+Zunächst erzeugen wir eine zusammenfassende Zählstatistik:
+
+
+```r
+count_summary <- title_idn_gnd_idn %>% 
+  summarise(n_title = n_distinct(idn_titel), n_terms = n_distinct(idn_sw))
+```
+
+In dem so erzeugten Datensatz gibt es 245 verschiedene GND-Terme,
```
Comment (niko2342) [inline docs/book/src/de/using-pica-rs-with-rmarkdown/using-pica-rs-with-rmarkdown.md:54]: Könntest du bitte im gesamten Tutorial "GND-Term" durch "GND-Entität" ersetzen.

**Review by niko2342 — CHANGES_REQUESTED** [review]
Lieber Max,

vielen Dank für das Tutorial. Ich habe noch ein paar Anmerkungen; danach kann der PR in den main-Branch.

BG
Nico

**Comment by mfakaehler** [comment]
Ich bin jetzt durch mit meinen Korrekturen.

**Pull request merged**